anime-list-master.xml

```diff
@@ -1574,7 +1574,7 @@
   <anime anidbid="373" tvdbid="77681" defaulttvdbseason="0" episodeoffset="" tmdbid="" imdbid="tt1796619">
     <name>Slayers Gorgeous</name>
     <mapping-list>
-      <mapping anidbseason="1" tvdbseason="0">;1-10;2-10;3-10;</mapping>
+     <mapping anidbseason="1" tvdbseason="0">;1-10;2-10;3-11;</mapping>
```

Comment:
accidentally removed a space from the front of the line

Reply:
In this case there is no complete movie on TVDB, so if you give TVDB priority but only have 1 video file, it will show "Slayers Gorgeous 1/2". It's better to map the first episode to 0.